packages/core/src/components/ui/ProductGallery/ProductGallery.tsx

```diff
@@ -54,7 +54,6 @@ export interface ProductGalleryProps {
       clearButtonLabel: FilterSliderProps['clearButtonLabel']
       applyButtonLabel: FilterSliderProps['applyButtonLabel']
     }
-    deliverySettings?: FilterSliderProps['deliverySettings']
```

Comment:
There isn't a `deliverySettings` in this section CMS config anymore, when this info was removed this code should've been deleted - that's why I've deleted it now